R/registry.R

```diff
@@ -202,6 +202,24 @@ list_registered_models <- function(max_results = 100, page_token = NULL, client
   res
 }
 
+parse_versions <- function(verions) {
```

Comment:
typo in the arg name here (it's tough to review on my phone)

Reply:
this is how you know i've been staring at Rstudio too much today